includes/wireguard.conf-template

```diff
@@ -0,0 +1,12 @@
+ce]
```

Comment:
That looks like it needs some help, did you mean `[Service]`

Reply:
[Interface]
yes it is untested so far so I didn't notice that